### PR TITLE
3 things have been changed.

### PR DESCRIPTION
**Pull request by Crispy13**
1. VAE branch output will be included model output. (VAE branch will be trained)
2. Minor change in dice score calculation (regardless of batch size)
3. SpatialDropout3D will be used instead of Dropout.

### CODE REVIEW

File: model.py [hunk 1]
```diff
@@ -107,83 +107,44 @@ def sampling(args):
...
-    return loss_
-
+    
+    intersection = K.sum(K.abs(y_true * y_pred), axis=[-3,-2,-1])
+    dn = K.sum(K.square(y_true) + K.square(y_pred), axis=[-3,-2,-1]) + 1e-8
+    return K.mean(2 * intersection / dn, axis=[0,1])
+
+
+def loss_gt(e=1e-8):
+    def loss_gt_(y_true, y_pred):
```
Comment (IAmSuyogJadhav) [inline model.py:139]: Please copy relevant parts of documentation strings from the original file and put them here (or write your own documentation strings) to make your functions informative and readable by others.

**Review by IAmSuyogJadhav — CHANGES_REQUESTED** [review]
I have put down a comment regarding documentation. Please do the requested minor changes.

**Comment by IAmSuyogJadhav** [comment]
Please confirm if you have tested these functions to work correctly. Are the model weights changing as expected after this change?

File: model.py (hunk 1 above)
Comment (Crispy13) [reply model.py:139]: I edited the doc strings part you mentioned. Please check this modifications.

**Comment by Crispy13** [comment]
> Please confirm if you have tested these functions to work correctly. Are the model weights changing as expected after this change?

Yes. The VAE branch is contained in the model and now trained well.

**Review by IAmSuyogJadhav — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**